WORKSPACE

```diff
@@ -132,15 +163,21 @@ workspace_refs(
 # Load NPM dependencies #
 #########################
 
+# Load //builder/nodejs
 load("@vaticle_dependencies//builder/nodejs:deps.bzl", nodejs_deps = "deps")
-nodejs_deps(["@vaticle_dependencies//builder/nodejs:remove-node-patches.patch"])
-load("@build_bazel_rules_nodejs//:index.bzl", "node_repositories", "yarn_install")
+nodejs_deps()
+
+load("@build_bazel_rules_nodejs//:repositories.bzl", "build_bazel_rules_nodejs_dependencies")
+build_bazel_rules_nodejs_dependencies()
 
+load("@build_bazel_rules_nodejs//:index.bzl", "node_repositories", "yarn_install")
 node_repositories(
-    preserve_symlinks = False,
+    node_version = "17.9.1"
 )
+
 yarn_install(
-  name = "npm",
-  package_json = "//grpc/nodejs:package.json",
-  yarn_lock = "//grpc/nodejs:yarn.lock",
-)
+    name = "npm",
+    package_json = "//grpc/nodejs:package.json",
+    yarn_lock = "//grpc/nodejs:yarn.lock",
```

Comment:
Interestingly, we use `npm` to install Bazel rules that will allow us to compile protobuf and services into Typescript!